devtools-ci-index.yaml

```diff
@@ -220,7 +220,12 @@
                 oc deploy {svc_name} --latest
               fi
               if [ "{template_name}" != "none" ]; then
-                oc process -f {template_name} | oc apply -f -
+                PARAMETERS=""
+                TAG=$(echo $GIT_COMMIT | cut -c1-6)
```

Comment:
Technically you could move TAG down into the if below so you only execute it if IMAGE_TAG is in the template